ui/app/components/identity/popup-metadata.js

```diff
@@ -0,0 +1,29 @@
+import Base from './_popup-base';
+import { computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+
+export default Base.extend({
+  model: alias('params.firstObject'),
+  key: computed('params', function() {
+    return this.get('params').objectAt(1);
```

Comment:
i thought i remembered us talking in a previous PR moving away from using `this.get` and instead accessing the property directly with `this.params`. is that the case? i don't fully understand the tradeoffs between the two, so any context you have would be great!

Reply:
Oops, yep - I copied this code from last June when I erroneously deleted it and that was before the ember upgrade - here's a link to the RFC that describes the change: https://github.com/emberjs/rfcs/blob/master/text/0281-es5-getters.md I'll update this file.

Reply:
hurray! thanks for the link -- very helpful!